petar-stoyanov/homework1/cv.css

```diff
@@ -0,0 +1,314 @@
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#fce8d3', GradientType=1 );
+	
+}
+
+#technical-skills h3 {
+	color: #ff9933;
+}
+#personal-skills{
+	background: rgba(255,255,255,1);
+	background: -moz-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(255,229,212,1) 100%);
+	background: -webkit-gradient(left top, right bottom, color-stop(0%, rgba(255,255,255,1)), color-stop(59%, rgba(255,255,255,1)), color-stop(100%, rgba(255,229,212,1)));
+	background: -webkit-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(255,229,212,1) 100%);
+	background: -o-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(255,229,212,1) 100%);
+	background: -ms-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(255,229,212,1) 100%);
+	background: linear-gradient(135deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(255,229,212,1) 100%);
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#ffe5d4', GradientType=1 );
+	
+}
+
+#personal-skills h3 {
+	color: #ff6600;
+}
+
+#personal-information{
+	background: rgba(255,255,255,1);
+	background: -moz-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(235,235,235,1) 100%);
+	background: -webkit-gradient(left top, right bottom, color-stop(0%, rgba(255,255,255,1)), color-stop(59%, rgba(255,255,255,1)), color-stop(100%, rgba(235,235,235,1)));
+	background: -webkit-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(235,235,235,1) 100%);
+	background: -o-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(235,235,235,1) 100%);
+	background: -ms-linear-gradient(-45deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(235,235,235,1) 100%);
+	background: linear-gradient(135deg, rgba(255,255,255,1) 0%, rgba(255,255,255,1) 59%, rgba(235,235,235,1) 100%);
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#ebebeb', GradientType=1 );
+	color: #black;
+}
+#education, #working-experience, #technical-skills, #personal-skills, #personal-information {
+	/*round*/
+	-webkit-border-radius: 5px;
+	-moz-border-radius: 5px;
+	border-radius: 5px;
+	/*shadow*/
+	-webkit-box-shadow: 2px 2px 5px 0px rgba(23,66,3,1);
+	-moz-box-shadow: 2px 2px 5px 0px rgba(23,66,3,1);
+	box-shadow: 2px 2px 5px 0px rgba(23,66,3,1);
+	/**/
+	border-width: 1px;
+	border-style: inset;
+	padding: 10px;
+	margin: auto;
+	margin: 10px;
+	margin-left: 80px;	
+	width: 90%;	
+}
+
+footer {
+	color: rgba(126,126,126,0.6);
+	background-color: #d9d9d9;
+	width: 100%;
+	font-size: 0.9em;
+	width: 100%;
+	float: right;
+	border: 1px solid black;
+}
+/*hr {
+	height: 0px;
+	border: 1;
+	border-color: green;
+}*/
+/*hr.bottom {
+	height: 0px;
+	border: 1;
+	border-color: green;
+}*/
+/*hr.top {
+	height: 0px;
+	border: 1;
+	border-color: green;				
+}*/
```

Comment:
@petar-stoyanov7 

Good job with the CV :+1:  The idea is creative and original : )